src/amo/components/AddonMoreInfo/styles.scss

```diff
@@ -0,0 +1,3 @@
+.Categories-list--justify-content {
+  justify-content: start;
```

Comment:
Is this really needed?

Reply:
No, for the simple design it is not required. Initially I implemented different design, so added this. I will remove this in the next update.